src/main/java/com/exasol/dbbuilder/objectwriter/ImmediateDatabaseObjectWriter.java

```diff
@@ -81,23 +82,34 @@ public void write(final Schema schema) {
         writeToObject(schema, "CREATE SCHEMA " + schema.getFullyQualifiedName());
     }
 
+    @Override
+    public void write(final Script script) {
+        final StringBuilder builder = new StringBuilder("CREATE SCRIPT " + script.getFullyQualifiedName());
+        builder.append(" AS\n");
+        builder.append(script.getContent());
+        builder.append("\n/");
+        writeToObject(script, builder.toString());
+    }
+
     @Override
     public void write(final Table table) {
-        final StringBuilder sqlBuilder = new StringBuilder("CREATE TABLE " + table.getFullyQualifiedName() + " (");
+        final StringBuilder builder = new StringBuilder("CREATE TABLE ");
+        builder.append(table.getFullyQualifiedName());
+        builder.append(" (");
         boolean first = true;
         for (final Column column : table.getColumns()) {
             if (first) {
                 first = false;
             } else {
-                sqlBuilder.append(", ");
+                builder.append(", ");
             }
-            sqlBuilder.append("\"");
-            sqlBuilder.append(column.getName());
-            sqlBuilder.append("\" ");
-            sqlBuilder.append(column.getType());
+            builder.append("\"");
+            builder.append(column.getName());
+            builder.append("\" ");
+            builder.append(column.getType());
         }
-        sqlBuilder.append(")");
-        writeToObject(table, sqlBuilder.toString());
+        builder.append(")");
+        writeToObject(table, builder.toString());
```

Comment:
This method is quite long, Can we extract something from here?

Reply:
Extracting here would not increase readability.

Changed to fluent builder and more compact separator check to make the code more compact.